forge/test/model_demos/high_prio/cnn/pytorch/test_yolox.py

```diff
@@ -76,6 +76,11 @@ def test_yolox_pytorch(variant, test_device):
     model = exp.get_model()
     ckpt = torch.load(f"{variant}.pth", map_location="cpu")
     model.load_state_dict(ckpt["model"])
+
+    # Set to false as it is part of model post-processing
+    # to avoid pcc mismatch due to inplace slice and update
+    model.head.decode_in_inference = False
```

Comment:
Can we add post-processing at the end of this test? I.e. to run it on host?

Reply:
Updated @nvukobratTT